paramiko/config.py

```diff
@@ -80,6 +81,27 @@ def parse(self, file_obj):
                 # at the end (for compatibility with issue #415). After 3.x, it
                 # will simply not get stripped, leaving a nice explicit marker.
                 host['config'][key] = None
+            elif key == 'include':
+                # support for Include directive in ssh_config
+
+                path = value
+                # according to SSH documentation if the path is relative, we look forward in ~/.ssh
+                if path[0] != '/' and path[0]!='~':
+                    path = '~/.ssh/'+path
+                #expand the user home path
+                path = os.path.expanduser(path)
+
+                if parsed_files == None:
+                    parsed_files = []
+
+                #parse every included file
+                for filename in glob.iglob(path):
+                    if os.path.isfile(filename):
+                        if path in parsed_files:
+                            raise Exception("Include loop detected in ssh config file: %s" %path)
+                        with open(filename) as fd:
+                            parsed_files.append(path);
+                            self.parse(fd,parsed_files)
```

Comment:
more minor style nits - space after comma, no semicolon at end of the previous line, spaces around `%` three lines up